src/components/form/form.jsx

```diff
@@ -128,3 +132,74 @@ Form.Feedback = ({ className, message }) => {
     </>
   );
 };
+
+Form.Combobox = ({ id, labelText, className, list, name, nameKey, idKey }) => {
+  const [selectedItems, setSelectedItems] = useState([]);
+  const [query, setQuery] = useState("");
+  const filteredList =
+    query === ""
+      ? list
+      : list.filter((item) => {
+          return item[nameKey].toLowerCase().includes(query.toLowerCase());
+        });
+  return (
+    <div className={`flex font-bold font-arial flex-col py-3 ${className}`}>
+      <label htmlFor={id} className={`mb-1.5 text-lg text-start `}>
+        {labelText}
+      </label>
+      <Combobox
+        value={selectedItems}
+        onChange={setSelectedItems}
+        multiple
+        nullable
+        name={name}
+      >
+        <div className=" relative font-normal py-3 px-4 border-solid rounded-none border border-black focus-within:ring-black focus-within:ring-1 flex gap-2.5 flex-wrap">
+          {selectedItems.length > 0 &&
+            selectedItems.map((item) => (
+              <div
+                key={item[idKey]}
+                className="flex gap-1.5 bg-neutral-300 px-1.5 py-1"
+              >
+                <span>{item[nameKey]}</span>
+                <button
+                  type="click"
+                  onClick={() => {
+                    setSelectedItems(
+                      selectedItems.filter(
+                        (element) =>
+                          element[idKey] !== item[idKey] ||
+                          element[nameKey] !== item[nameKey]
+                      )
+                    );
+                  }}
+                >
+                  <img src={cross} />
+                </button>
+              </div>
+            ))}
+          <Combobox.Input
+            onChange={(event) => setQuery(event.target.value)}
+            value={query}
+            className="font-nomral border-none focus:outline-none flex-grow flex-shrink-0 w-16"
+          />
+          <Combobox.Button className="absolute right-5 flex top-4">
+            <img src={down} className="h-2.5 w-2.5" />
+          </Combobox.Button>
+        </div>
+        <Combobox.Options>
```

Comment:
When there are a lot of options, the combobox options just continue to push the page down. Without solving for this directly I'd have to play with solutions but I'd image you would need to display the options within a div that is positioned absolutely over the rest of the page. Additionally, this div should have a max height and overscroll-auto so that the user can scroll through the options instead having one long list.